### PR TITLE
Adding checklists to board

### PR DESCRIPTION
**Pull request by gvieira**
Hi there,

This adds the option to get all the checklists from a given board. Simple stuff, really.

Please let me know if we need any further adjustments.

Cheers,
G.V.

### CODE REVIEW

File: spec/board_spec.rb [hunk 1]
```diff
@@ -225,6 +225,19 @@ module Trello
       end
     end
 
+    context "checklists" do
+      before do
+        allow(client)
+          .to receive(:get)
+          .with("/boards/abcdef123456789123456789/checklists", { filter: :all})
```
Comment (joneslee85) [inline spec/board_spec.rb]: nitpick, remove the empty space after `{`

**Comment by joneslee85** [comment]
LGTM

File: spec/board_spec.rb (hunk 1 above)
Comment (gvieira) [inline spec/board_spec.rb]: Done! Thanks for reviewing.

**Comment by gvieira** [comment]
Sorry to bother, but is there any chance you could merge this? I would really make use of this adjustment.

**Pull request merged**